target/linux/meson/patches-5.15/904-usb-disable-ACA-check.patch

```diff
@@ -0,0 +1,18 @@
+--- a/drivers/phy/amlogic/phy-meson8b-usb2.c
```

Comment:
why this is required, add a patch header to describe it, and you should submit this to upstream kernel

this patch should be numbered as 700